examples/logical_switch.rb

```diff
@@ -9,10 +9,14 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
+# NOTE 1: This example requires two Scopes named "Scope1" and "Scope2" to be present in the appliance.
+# NOTE 2: The api_version client should be greater than 200 if you run the examples using Scopes
```

Comment:
I don't like saying it that way, maybe you should try using "should be 300 or greater". 

Why? If for some reason an API 201 is created this comment would be wrong.